apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/User/UserNode.cs

```diff
@@ -13,21 +13,26 @@
 // limitations under the License.
 
 using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
-using Energinet.DataHub.WebApi.GraphQL.Resolvers;
 
-namespace Energinet.DataHub.WebApi.GraphQL.Types.User;
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.User;
 
-public class GetUserResponseType : ObjectType<GetUserResponse>
+[GraphQLName("User")]
+[ObjectType<GetUserResponse>]
```

Comment:
question: does this actually work? it still looks like it is called `GetUserResponse` in the schema snapshot. maybe if you add it to your partial `GetUserResponse` class, but i think i would still prefer a descriptor configuration in this file.

Reply:
You are so right, fixed now